### ISSUE TITLE
LRUCache is not transpiled in ESM version, so it fails uglifyJS build.

### ISSUE THREAD

**Issue by supermoos** [kuitos/axios-extensions#40]
The included LRUCache uses const's, which fails when I try to process it trough Uglify - could we use a version that's properly transpiled?

```
Unexpected token: keyword (const) [./node_modules/axios-extensions/node_modules/lru-cache/index.js:4,0][main.legacy.js:7204,0]
Details:
    domain: [object Object]
    domainThrown: true
```

**Comment by kuitos** [comment]
As lru-cache not works well in react-native environment below v5.0.0, so we need to upgrade our dependency. If you need your bundle target to es5, you might make your compiler transpile the lru-cache package as well.
example with babel:
```json
{
  include: ['/node_modules/lru-cache']
}
```

**Comment by supermoos** [comment]
I ended up just using v.3.0.3

**Comment by kuitos** [comment]
released v3.0.5
Thanks for your feedback!
